pages/[id]/bridge.tsx

```diff
@@ -101,20 +132,66 @@ const Bridge = () => {
   };
 
   const handleBridge = () => {
-    return bridgeTarget === BridgeTarget.L2 ? bridgeToL2!() : bridgeToL1!();
+    if (bridgeTarget === BridgeTarget.L2) {
+      if (!bridgeToL2) throw new Error('bridgeToL2 is not defined');
+      bridgeToL2();
+    } else {
+      if (!bridgeToL1) throw new Error('bridgeToL1 is not defined');
+      bridgeToL1();
+    }
   };
 
-  const source =
-    bridgeTarget === BridgeTarget.L2
-      ? { ...l1Config, ...l1, fee: fees?.l1 || BigInt(0) }
-      : { ...l2Config, ...l2, fee: fees?.l2 || BigInt(0) };
-  const target =
+  const formatError = (e: Error | null): ErrorReturnType => {
+    if (e === null) return { errorType: undefined, errorReason: undefined };
+    const errorTypes = [
+      {
+        errorType: ErrorType.UserRejectedRequest,
+        errorSearchString: 'User rejected the request.',
+        prettyReason: `Error: User rejected the request.`,
+      },
+      {
+        errorType: ErrorType.ERC20AmountError,
+        errorSearchString: 'ERC20: transfer amount exceeds balance',
+        prettyReason: `Error: Not enough ${source.token?.symbol} in wallet.`,
+      },
+      {
+        // Here we duplicate the ERC20AmountError -- this is because the error search string is slightly different for L1 and L2,
+        // but they're functionally equivalent.
+        errorType: ErrorType.ERC20AmountError,
+        errorSearchString: 'ERC20: burn amount exceeds balance',
+        prettyReason: `Error: Not enough ${source.token?.symbol} in wallet.`,
+      },
+      {
+        errorType: ErrorType.ETHError,
```

Comment:
This error name is a little vague. Maybe, InsufficientNativeCurrencyError

Reply:
thanks, done!